deep_groove_project/src/first_eda/first_eda.py

```diff
@@ -42,10 +43,10 @@ def show_3d_scatter(df):
     n = df.iloc[:, 1]
     lifetime = df.iloc[:, 2]
 
-    ax.scatter(fr, n, lifetime)
+    ax.scatter(fr, n, np.log(lifetime))
     ax.set_xlabel('Fr')
     ax.set_ylabel('n')
-    ax.set_zlabel('Lifetime')
+    ax.set_zlabel('log(Lifetime)')
```

Comment:
It is good practice to provide the phyiscal units. "Lifetime in h"

Reply:
added the value to the Label

Reply:
I also provided physical units for the other parameters